uint16 degrees
uint8 time
---
bool exec_status